README.md

```diff
@@ -88,6 +89,17 @@ $ conda install -c conda-forge optuna
 Optuna supports Python 3.5 or newer.
 
 
+## Communication
+
+- [Github Issues] for bug reports, feature requests, questions, and etc.
```

Comment:
Maybe you wrote `etc` because I mentioned it earlier, sorry if that was the case. I believe we can tidy it up a bit as follows (assuming it's okay not to be exhaustive as for the "documentation" template).
```suggestion
- [Github Issues] for bug reports, feature requests and questions.
```